modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -12,25 +12,19 @@ In the table, features are marked with the following statuses:
 * *GA*: _General Availability_
 
 |===
-|*OpenShift GitOps* 7+|*Component Versions*|*OpenShift Versions*
+|*OpenShift GitOps* 8+|*Component Versions*|*OpenShift Versions*
 
-|*Version*|*kam*|*Helm*|*Kustomize*|*Argo CD*|*ApplicationSet*|*Dex*||
-|1.6.0 |0.0.46 TP |3.8.1 GA |4.4.1 GA |2.4.5 GA |2.4.5 GA |2.30.3 GA |4.8-4.11 |GA
-|1.5.0 |0.0.42 TP |3.8.0 GA |4.4.1 GA |2.3.3 GA |0.4.1 TP |2.30.3 GA |4.8-4.11 |GA
-|1.4.0 |0.0.41 TP |3.7.1 GA |4.2.0 GA |2.2.2 GA |0.2.0 TP |2.30.0 GA |4.7-4.11 |GA
-|1.3.0|0.0.40|3.6.0|4.2.0|2.1.2|0.2.0|2.28.0|4.7-4.9|GA
-|1.2.0|0.0.38|3.5.0|3.9.4|2.0.5|0.1.0|N/A|4.8|GA
-|1.1.0|0.0.32|3.5.0|3.9.4|2.0.0|N/A|N/A|4.7|GA
```

Comment:
**Question:** Any particular reason why these rows are removed?

**Suggestion:** Retain these in all the OCP versions of GitOps RN

Reply:
new issue created to track this comment: https://issues.redhat.com/browse/RHDEVDOCS-4958